Fix:

error: implicit declaration of function 'check_perms_in_inode' [-Werror,-Wimplicit-function-declaration]
--- src/ext2fs.h	2008-06-26 00:34:50.000000000 -0500
+++ src/ext2fs.h	2024-05-16 10:05:06.000000000 -0500
@@ -87,6 +87,8 @@
 
 int check_perms(perms_struct, ext2_ino_t ino,
                 int perms_requested);
+int check_perms_in_inode(perms_struct ps, struct ext2_inode *inode,
+                         int perms_requested);
 int set_perms(perms_struct ps, struct ext2_inode *inode);
 
 /* ext2fs.c */
